modular_skyrat/modules/nullf­a­g_masspr/README.txt

```diff
@@ -0,0 +1,16 @@
+NullF­a­g here
```

Comment:
delet

Reply:
don't screech at me because "funny owner doesn't know how to use the config" then